pyflakes/test/test_api.py

```diff
@@ -148,8 +149,7 @@ def tearDown(self):
     def makeEmptyFile(self, *parts):
         assert parts
         fpath = os.path.join(self.tempdir, *parts)
-        fd = open(fpath, 'a')
-        fd.close()
+        open(fpath, 'a').close()
```

Comment:
this is new to me - why is this helpful?

Reply:
it's no different than the left hand side, but reduces the scope in which an unclosed file descriptor could leak (notice that I just removed the temporary variable)

Reply:
im not sure that it changes possibility of leaks, but happy to merge anyway - its nice to read it this way.  I was just hoping to understand it better

Reply:
it's more to prevent a future modification from inserting code between the two statements (an improvement of maintenance, not of functionality)